### ISSUE TITLE
China Securities Regulatory Commission (CSRC) Administrative Penalties

### ISSUE THREAD

**Issue by Kaldea** [opensanctions/crawler-planning#274]
### Data URL

http://www.csrc.gov.cn/csrc/c100028/common_list.shtml

### Publisher

China Securities Regulatory Commission

### Publisher country/territory code

cn

### Type of data

Sanctions (Governments barring specific engagement e.g. financial transaction with these entities)

### Coverage region

region:Asia/Pacific

### Can you tell us more?

Main Administrative Penalties Page: http://www.csrc.gov.cn/csrc/c100028/common_list.shtml
Enforcement Actions and Notices: http://www.csrc.gov.cn/csrc/c100026/common_list.shtml

The CSRC publishes detailed administrative penalties and enforcement actions against individuals and organizations violating securities laws in China. This includes names, identification details, the nature of the violation, and the penalties imposed. Incorporating this data would enhance coverage of regulatory actions from China, which is significant given the global focus on AML and compliance related to Chinese entities.

### This is a suggestion or request

- [X] I understand.

**Comment by jbothma** [comment]
yeah - tricky to find, and the site is slow. The penalties I've found so far were free text - might be able to parse a main subject name out at least:

Administrative penalties http://www.csrc.gov.cn/csrc/c101971/zfxxgk_zdgk.shtml linked from http://www.csrc.gov.cn/
![image](https://github.com/user-attachments/assets/d309c15b-374d-47f2-85b2-fcdf3a9bfd92)

![image](https://github.com/user-attachments/assets/145f14a5-f651-4bcc-b26c-3e323195477a)
![image](https://github.com/user-attachments/assets/52ccdd41-6901-4088-9549-00a4da821f00)

 and

![image](https://github.com/user-attachments/assets/1c493329-7ff0-472e-8d39-6d08932d0e81)

**Comment by brrttwrks** [comment]
This source doesn't contain a readily usable list and the information in free-form is inconsistent. While it would be in-scope and legal, it is not feasible.
